src/autocapture-utils.ts

```diff
@@ -303,14 +303,18 @@ export function isAngularStyleAttr(attributeName: string): boolean {
     return false
 }
 
-export function loadScript(scriptUrlToLoad: string, callback: (event: Event) => void): void {
+export function loadScript(
+    scriptUrlToLoad: string,
+    callback: (event: Event) => void,
+    restrictToHTMLBody?: boolean
+): void {
     const scriptTag = document.createElement('script')
     scriptTag.type = 'text/javascript'
     scriptTag.src = scriptUrlToLoad
     scriptTag.onload = callback
 
     const scripts = document.getElementsByTagName('script')
```

Comment:
```suggestion
    const scripts = document.querySelectorAll('body > script')
```

Chang this line, and I think everything else will work without any other changes. I don't think we need to provide a new parameter, as by default script tags do get added into body.